### PR TITLE
Update Docs

### PR DESCRIPTION
**Pull request by jeff-hernandez**
This PR will make the following updates:
- add guide for using compose with featuretools
-  add guide for using label transforms
- update diagram in main concepts

### CODE REVIEW

File: docs/source/guides/using_compose_with_featuretools.ipynb [hunk 1]
```diff
@@ -563,7 +563,7 @@
     "Describe Labels\n",
     "---------------\n",
     "\n",
-    "After transforming the labels, we could use :meth:`LabelTimes.describe` to print out the distribution with settings and transforms used to make the labels. This is useful as a reference for understanding how the labels were generated from raw data. We could also see the balance in the labels from the distribution."
+    "After transforming the labels, we could use :meth:`LabelTimes.describe` to print out the distribution with the settings and transforms that were used to make the labels. This is useful as a reference for understanding how the labels were generated from raw data. Also, the label distribution is helpful for determining if we have disproportionate labels."
```
Comment (kmax12) [inline docs/source/guides/using_compose_with_featuretools.ipynb]: maybe imbalanced is better than "disproportionate"?

File: docs/source/guides/using_label_transforms.ipynb [hunk 2]
```diff
@@ -12,7 +12,7 @@
     "Using Label Transforms\n",
     "======================\n",
     "\n",
-    "In this guide, we will demonstrate how to use the transforms that are available on :class:`LabelTimes`.\n",
+    "In this guide, we will demonstrate how to use the transforms that are available on :class:`LabelTimes`. Each transform will return a copy of the label times. This is useful for trying out different transforms without having to recalculae the labels.\n",
```
Comment (kmax12) [inline docs/source/guides/using_label_transforms.ipynb]: typo in "recalculae"

Comment (jeff-hernandez) [reply docs/source/guides/using_label_transforms.ipynb]: fixed typo

File: docs/source/guides/using_compose_with_featuretools.ipynb (hunk 1 above)
Comment (jeff-hernandez) [reply docs/source/guides/using_compose_with_featuretools.ipynb]: yes, made the update

File: docs/source/install.rst [hunk 3]
```diff
@@ -0,0 +1,7 @@
+=======
+Install
+=======
+
+Install ComposeML by using :code:`pip`::
```
Comment (kmax12) [inline docs/source/install.rst]: can you add what python versions are supported?

File: docs/source/api_reference.rst [hunk 4]
```diff
@@ -1,3 +1,4 @@
+=============
 API Reference
 =============
```
Comment (kmax12) [inline docs/source/api_reference.rst]: can we add the different label time transforms, plotting methods, etc to the api reference?

File: docs/source/api_reference.rst [hunk 5]
```diff
@@ -1,11 +1,48 @@
...
+
+.. autosummary::
+    :nosignatures:
+
+    LabelTimes.apply_lead
+    LabelTimes.bin
+    LabelTimes.describe
+    LabelTimes.threshold
+
+Plotting Methods
```
Comment (jeff-hernandez) [inline docs/source/api_reference.rst]: Placeholder table for plotting methods while restructuring.

File: docs/source/install.rst (hunk 3 above)
Comment (jeff-hernandez) [reply docs/source/install.rst]: Added the supported python version in [this](https://github.com/FeatureLabs/compose-ml/pull/29/commits/dc08c50ee375ebe6532473147c89df8edadda34e#diff-a2e1e3b391e0e24fcd7e517d40d9bba8R5) commit

File: docs/source/api_reference.rst (hunk 4 above)
Comment (jeff-hernandez) [reply docs/source/api_reference.rst]: Added tables to API reference in [this](https://github.com/FeatureLabs/compose-ml/pull/29/commits/6cb5017ff13fe2d9d8814512caa50ceabb4c9cf4#diff-18294892d540ca2ec3e466841181ff63R27) commit.

File: docs/requirements.txt [hunk 6]
```diff
@@ -1,4 +1,4 @@
-git+https://github.com/FeatureLabs/compose-ml.git
+git+https://github.com/FeatureLabs/compose-ml.git@update-docs
```
Comment (kmax12) [inline docs/requirements.txt]: should this be updated?

Comment (jeff-hernandez) [reply docs/requirements.txt]: Yes

**Pull request merged**